pkgs/applications/science/biology/sumatools/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, fetchFromGitLab, gcc, zlib }:
+
+let
+  nativeBuildInputs = [ gcc ];
+  meta = with stdenv.lib; {
+    description = "Fast and exact comparison and clustering of sequences";
+    homepage = http://metabarcoding.org/sumatra;
+    maintainers = [ maintainers.bzizou ];
+    platforms = platforms.unix;
+  };
+
+in rec {
+
+  # Suma library
+  sumalibs = stdenv.mkDerivation rec {
+      version = "1.0.34";
+      pname = "sumalibs";
+      src = fetchFromGitLab {
+        domain = "git.metabarcoding.org";
+        owner = "obitools";
+        repo = pname;
+        rev = "sumalib_v${version}";
+        sha256 = "0hwkrxzfz7m5wdjvmrhkjg8kis378iaqr5n4nhdhkwwhn8x1jn5a";
+      };
+      makeFlags = "PREFIX=$(out)";
+  };
+
+  # Sumatra
+  sumatra = stdenv.mkDerivation rec {
+      version = "1.0.34";
```

Comment:
Please consistently use 2 spaces for indentation

Reply:
Sorry for that :-)

Reply:
@bzizou the text within sumalibs, sumatra, and sumaclust still has 4 spaces of indentation instead of 2.